docs/onboarding/qsv-pro.mdx

```diff
@@ -0,0 +1,48 @@
+---
+sidebar_position: 6
+---
+
+# ⚙️ qsv pro Setup
+
+## ℹ About qsv pro
+
+Welcome to qsv pro, a powerful desktop app designed for spreadsheet data wrangling. Built on the foundation of the [qsv CLI tool](/qsv/qsv-cli), qsv pro offers a feature-packed local GUI for an enhanced user experience.
+
+## 🏗 Built with
```

Comment:
The "Built with" section may be unnecessary

Reply:
If it doesn't feel right, let's discontinue with the next commit (which is soon, I'm working on stuff from analyze to be added here too), especially since it's meant for interns to understand the technologies we're using to build qsv pro